source/config/configFlags.py

```diff
@@ -220,3 +220,23 @@ def _displayStringLabels(self):
 			# Translators: This is a label for the automatic update behaviour for add-ons.
 			self.DISABLED: _("Disabled"),
 		}
+
+@unique
+class OutputMode(DisplayStringIntEnum):
+	OFF = 0
+	SPEECH = 1
+	BRAILLE = 2
+	SPEECH_AND_BRAILLE = 3
```

Comment:
```suggestion
class OutputMode(DisplayStringIntFlag):
	OFF = 0b0
	SPEECH = 0b01
	BRAILLE = 0b10
	SPEECH_AND_BRAILLE = 0b11
```

or

```suggestion
class OutputMode(DisplayStringIntFlag):
	OFF = 0
	SPEECH = 1
	BRAILLE = 2
	SPEECH_AND_BRAILLE = SPEECH ^ BRAILLE
```

Reply:
I've ended up going for a bit of a hybrid approach here, as using a `Flag` subclass introduces problems with iterating over enum members.